.eslintrc.js

```diff
@@ -191,5 +162,53 @@ module.exports = {
         'react/prop-types': 'off',
       },
     },
+    {
+      files: ['*.d.ts'],
+      rules: {
+        'import/export': 'off', // Not sure why it doesn't work
+      },
+    },
+    {
+      files: ['*.tsx'],
+      rules: {
+        'no-restricted-imports': [
```

Comment:
This looks duplicated from the base rules.

Reply:
It allows an extra level of depth import to solve the Breakpoint problem.

Reply:
Yeah but that isn't correct. We should only be testing public types. If we don't have a public type for breakpoints then this is a problem.

Reply:
Agree, we need to fix that